saleor/graphql/metrics.py

```diff
@@ -0,0 +1,30 @@
+from contextlib import AbstractContextManager
+
+from ..core.telemetry import AttributeValue, MetricType, Unit, meter
+
+# Initialize metrics
+meter.create_metric(
+    "saleor.graphql_queries",
+    service_scope=True,
+    type=MetricType.COUNTER,
+    unit=Unit.REQUEST,
+    description="Number of GraphQL queries.",
+)
+meter.create_metric(
+    "saleor.graphql_query_duration",
+    service_scope=True,
+    type=MetricType.HISTOGRAM,
+    unit=Unit.MILLISECOND,
+    description="Duration of GraphQL queries.",
+)
+
+
+# Helper functions
+def incr_graphql_queries(amount: int = 1) -> None:
```

Comment:
I was also thinking about some naming convention for the functions that record metrics. Since meters use "record", maybe we could also name all these helpers using the `record_` prefix:
- `record_graphql_queries_count`
- `record_graphql_query_duration`

Reply:
Fixed